docs/UserGuide.md

```diff
@@ -81,13 +78,13 @@ If you need to switch to a new device, you can also export and import your previ
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`, `clear` and `switch`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit`, `clear`, `switch`, `import`, and `export`) will be ignored.<br>
```

Comment:
I think can remove import and export from here as they do take in extra parameters

Reply:
oh yess i forgot the path parameter hahaha thanks